ansible/roles/test/tasks/ptf_runner_reboot.yml

```diff
@@ -41,6 +41,18 @@
       vars:
         supervisor_host: "{{ ptf_host }}"
 
+    - include_vars: "vars/topo_{{testbed_type}}.yml"
+
+    - name: Expand properties into props
+      set_fact: props="{{configuration_properties['common']}}"
+      when: testbed_type in ['t0', 't0-56', 't0-64', 't0-116', 't0-64-32']
+
+    # Generate route file
+    - name: Generate route-port map information
```

Comment:
can you also please update the pytest version of this test case? Similar template work is done [here](https://github.com/Azure/sonic-mgmt/blob/master/tests/common/fixtures/advanced_reboot.py#L239). The paramter to the ptfrunner is [here](https://github.com/Azure/sonic-mgmt/blob/master/tests/common/fixtures/advanced_reboot.py#L433)